src/main/java/com/gluonhq/substrate/util/ios/InfoPlist.java

```diff
@@ -361,4 +363,22 @@ private void verifyAssetCatalog(Path resourcePath, String platform, String minOS
         }
     }
 
+    private void copyOtherAssets(Path resourcePath) throws IOException {
```

Comment:
add some doc: this is copying $project/src/main/resources/assets/foo to $project/build bar?